public/index.tsx

```diff
@@ -31,15 +29,21 @@ if (locationUrl.hash !== "" || isEncodedPathUrl(locationUrl)) {
   window.history.replaceState(null, "", newRelativePath);
 }
 
-const routes = [
+// TODO these components are now lazy loaded. Should they get `Suspense`s around them? What should we fall back to?
+const routes: RouteObject[] = [
   {
     path: "/",
-    element: <LandingPage />,
+    lazy: async () => ({ Component: (await import("../website/components/LandingPage")).default }),
```

Comment:
Just curious, were you able to test at all that the lazy loading worked and the bundle size was kept smaller?

Reply:
I definitely confirmed that (`console.log`s I dropped inside of) scripts behind this lazy load don't run until that route actually loads. I did my best to root around the built files to work out where everything went and whether code was getting split properly, but wasn't able to 100% convince myself that way.

Reply:
Okay, great! Very nice :>